test/libsolidity/syntaxTests/pragma/broken_version_7.sol

```diff
@@ -1,3 +1,3 @@
 pragma solidity >0.5.0<;
 // ----
-// ParserError 1684: (0-24): Found version pragma, but failed to parse it. Please ensure there is a trailing semicolon.
+// ParserError 1684: (0-24): Invalid token in version Pragma: �
```

Comment:
Where does this `ÿ` come from? Maybe an uninitialized variable?

Reply:
I think it is not worth to implement a solution to cover such a corner case. What do you think?

Reply:
Is it that hard to fix though? We should simply issue a different error when we reach the end - that we expected something more but the pragma suddenly ended.

Reply:
Yes, I guess I wasn't clear. I meant that I don't think it is worth to implement a solution to still print the offending char/token... but definitely, should produce an error message.